cfme/tests/intelligence/reports/test_reports_manual.py

```diff
@@ -260,3 +260,129 @@ def test_reports_online_vms():
         1504010
     """
     pass
+
+
+@test_requirements.report
+@pytest.mark.manual
+@pytest.mark.tier(2)
+@pytest.mark.meta(coverage=[1743579])
+def test_created_on_time_report_field():
+    """
+    Bugzilla:
+        1743579
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: medium
+        initialEstimate: 1/2h
+        setup:
+            1. Add a provider and provision a VM
+        testSteps:
+            1. Create a report with `Created on Time` field.
+        expectedResults:
+            1. `Created on Time` field column must not be empty for the recently created VM.
+    """
+    pass
+
+
+@test_requirements.report
+@pytest.mark.manual
+@pytest.mark.tier(2)
+@pytest.mark.meta(coverage=[1741588])
+def test_case_insensitive_report_filtering():
+    """
+    Bugzilla:
+        1741588
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: medium
+        initialEstimate: 1/2h
+        setup:
+            1. Queue a report based on VMs and Instances(make sure it is not empty).
+            2. Filter the report with `VM Name` - case exactly matching the subject VM name
+                and note the result.
+            3. Filter the report with `VM Name` - case not matching the subject VM name
+                and note the result.
+        testSteps:
+            1. Compare both the results.
+        expectedResults:
+            1. Both the results must be same.
+    """
+    pass
+
+
+@test_requirements.report
+@pytest.mark.manual
+@pytest.mark.tier(2)
+@pytest.mark.meta(coverage=[1725142])
+def test_reports_service_unavailable():
+    """
+    Bugzilla:
+        1725142
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: medium
+        initialEstimate: 1/2h
+        setup:
+            1. Load customer database on the appliance.
+        testSteps:
+            1. Navigate to Reports.
+            2. Navigate to `Details` page of the report with >31k rows.
+        expectedResults:
+            1. Reports must be accessible, there should be no 503 service unavailable error.
+            2. Details page must be accessible.
+    """
+    pass
+
+
+@pytest.mark.ignore_stream("5.10")
+@test_requirements.report
+@pytest.mark.manual
+@pytest.mark.tier(2)
+@pytest.mark.meta(coverage=[1714197])
+def test_optimization_reports():
+    """
+    Bugzilla:
+        1714197
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: high
+        initialEstimate: 1/2h
+        setup:
+            1. Navigate to Overview > Optimization.
```

Comment:
This to me seems like a test-step rather than setup, as there is no setup. What do you think?